OpenRA.Mods.Common/ServerTraits/MasterServerPinger.cs

```diff
@@ -64,7 +65,7 @@ public void Tick(S server)
 
 		public void ServerStarted(S server)
 		{
-			if (!server.Ip.Equals(IPAddress.Loopback) && LanGameBeacon != null)
+			if (!server.Address.Host.Equals("localhost") && LanGameBeacon != null)
```

Comment:
You might want to do a reverse dns lookup to catch all localhost addresses.

Reply:
The reverse lookup can stall for hundreds of milliseconds (see comment of pchote above) if it doesn't match localhost. It would be a one time effect for this case though.